<ui version="4.0" >
 <class>CDistributeDialog</class>
 <widget class="QDialog" name="CDistributeDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Distribute ACF</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="AcfDist.qrc" >:/Resources/Icons/Logo.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Files</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="2" column="0" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>License File</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>Distribute Directory</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>ACF Directory</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" >
       <widget class="QPushButton" name="LicenseFileBrowseButton" >
        <property name="text" >
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="AcfDirEdit" >
        <property name="text" >
         <string>../../..</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="DistributeDirEdit" />
      </item>
      <item row="1" column="2" >
       <widget class="QPushButton" name="DistributeDirBrowseButton" >
        <property name="text" >
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QPushButton" name="AcfDirBrowseButton" >
        <property name="text" >
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <widget class="QLineEdit" name="LicenseFileEdit" >
        <property name="text" >
         <string>../../../License.txt</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="title" >
      <string>Options</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="1" >
       <widget class="QCheckBox" name="ApplyLicenseCheck" >
        <property name="text" >
         <string>Apply License</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="DistTutorialCheck" >
        <property name="text" >
         <string>Distribute Tutorials</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" >
       <widget class="QCheckBox" name="DistDocCheck" >
        <property name="text" >
         <string>Distribute Documentation</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QCheckBox" name="DistSourcesCheck" >
        <property name="text" >
         <string>Distribute Sources</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::NoButton</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="AcfDist.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDistributeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>253</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDistributeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>321</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ApplyLicenseCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>LicenseFileBrowseButton</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>96</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel" >
     <x>368</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ApplyLicenseCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>LicenseFileEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>126</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel" >
     <x>181</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
